src/generate-report.ts

```diff
@@ -148,16 +149,26 @@ class GenerateCtrfReport implements Reporter {
   }
 
   extractFailureDetails(testResult: AssertionResult): Partial<CtrfTest> {
+    const messageStackTracePattern = /^\s{4}at/mu
+    const colorCodesPattern = /\x1b\[\d+m/gmu
```

Comment:
Disable eslint for this line by adding below comment to line above. 

// eslint-disable-next-line no-control-regex

Dealing with ANSI escape codes for coloring text in terminal outputs is a valid use case.